### DISCUSSION TITLE
docs: `additionalProperties: [Object object]` shown on some sections

### DISCUSSION THREAD

**Discussion by jamietanna** [renovatebot/renovate#30514] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

None

### Please tell us more about your question or problem

Ie on https://docs.renovatebot.com/self-hosted-configuration/#productlinks we see the `additionalProperties` not showing correctly

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
Seems to be the case for most options with `addtionalProperties` field.

Converted to issue: https://github.com/renovatebot/renovate/issues/30515
